src/framework/ui/viewPager/viewPager.component.tsx

```diff
@@ -190,7 +190,8 @@ export class ViewPager extends React.Component<ViewPagerProps> implements PanRes
 
   public scrollToIndex(params: { index: number, animated?: boolean }) {
     const { index, ...rest } = params;
-    const offset: number = this.contentWidth * index;
+    const childCount = this.getChildCount() - 1;
```

Comment:
let's get back to it in another pull request

Reply:
alright